### DISCUSSION TITLE
🎉 New Release: Settings UX Improvements (v1.8)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#86] [Announcements]
The v1.8 release includes some quality-of-life improvements in Note Toolbar’s Settings.

Update via Community Plugins.

## Improvements 🎉

**Settings:**

- Drag and drop toolbar items.
  - On mobile, you should feel a small vibration when you pick up an item, and another each time you drag it between rows in the list.
- Items in toolbars are shown in a condensed list view; clicking each item opens a form for further editing.
  - This list is formatted similar to the mobile menu: If a label is not set for an item, the list show the tooltip instead.
  - On mobile, a pencil icon is added to item previews so it’s a little more apparent that rows are tappable for editing.

![CleanShot 2024-06-11 at 11 37 20](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/735cdfa7-0706-4588-bab2-bfd4d5721c7a)

- Drag and drop folder mappings.
  - Moved (and changed) the delete icon to the left-hand side of the list to be consistent with Obsidian's settings (and to be further from where the drag icon is).

![CleanShot 2024-06-11 at 11 37 30](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/2d46136c-0836-4c90-a7c2-0f1d4606a57d)

- Added a button to collapse/expand toolbars and toolbar items from view.
  - This makes it easier to skip down to other sections to make changes.
  - Toolbars/Items are expanded by default, and the heading shows a count when collapsed.

![CleanShot 2024-06-11 at 11 37 10](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/57108806-50ae-4046-b089-4ff9b8306cf4)

## What's Next?

Check out my thoughts on [the immediate roadmap](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/85).

**Comment by FelipeRearden** [comment]
Working perfect on version 1.8.0-beta-10 on Obsidian Mobile on iPadOS with mouse and keyboard. 

✅ Drag and drop toolbar items.
✅ Drag and drop folder mappings.
✅ Added a button to collapse/expand toolbars and toolbar items from view.

All Settings Drag and Drop interactions are working perfect on Mobile using a mouse. To be honest, this is the best drag and drop interaction using mouse that I ever saw in Obsidian. It just works!

**Reply by chrisgurney** [reply]
@FelipeRearden Thank you so much for testing! 

It took a lot of tweaking to get the UX just right and I don't feel like I'm quite done... but I've got to get it out the door. 😅

**Reply by chrisgurney** [reply]
Now officially released. Thanks again for help with testing @FelipeRearden !

**Reply by FelipeRearden** [reply]
Just to let you know that I did a conceptual change in my vault and I am starting NTB from the ground on version 1.8.6.

I am stressing a lot the plugin one, keyboard navigation, Settings, CSS and complex commands and workspaces switches. 

Everything is going well. I let you know if I found something. 🙏

It is amazing to see where we are since the first public beta. What an amazing plugin 💙
